api/python/flaschen.py

```diff
@@ -34,12 +56,8 @@ def __init__(self, host, port, width, height, layer=5, transparent=False):
     self.transparent = transparent
     self._sock = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
     self._sock.connect((host, port))
-    header = ''.join(["P6\n",
-                      "%d %d\n" % (self.width, self.height),
-                      "255\n"]).encode('utf-8')
-    footer = ''.join(["0\n",
-                      "0\n",
-                      "%d\n" % self.layer]).encode('utf-8')
+    header = _HEADER_P6 % {b"width": self.width, b"height": self.height}
+    footer = _FOOTER_FT % {b"x": 0, b"y": 0, b"z": self.layer}
```

Comment:
I think these days we should not use the footer anymore but right awya the HEADER_P6_FT.
The initial version of the protocol was using the footer, but I think we can modernize this.
(not necessarily in this PR, can be seaparate).

Reply:
I've updated it to use the new header.  I was holding back in case changing it might cause issues.